### DISCUSSION TITLE
Handling blanks in CSVs

### DISCUSSION THREAD

**Discussion by blink-twice** [githubocto/flat-action#34] [Q&A] [answered]
I tried Flat with [my repository](https://github.com/IDFC-Institute/mumbai-covid-data). My input is a [published Google Sheet formatted as a CSV](https://docs.google.com/spreadsheets/d/e/2PACX-1vQjJsxljCc4p6R9bJXd0vCaYFeLNV_Mcb7ZSRVEw6DAwoSFYwffAvk5YlQMg_dCnZzVgFvvte8AzW6E/pub?gid=0&single=true&output=csv). The [Flat viewer page](https://flatgithub.com/IDFC-Institute/mumbai-covid-data/blob/main/mumbai_c19_age_wise_mortality.csv) seems to fill in zeros for blanks. (27 Dec 2020, 18 Dec 2020, etc.)  Same in the downloaded CSV from the Flat Viewer page. But the [CSV file](https://github.com/IDFC-Institute/mumbai-covid-data/blob/main/mumbai_c19_age_wise_mortality.csv) that Flat action creates in my repository contains blanks (as expected).

Is this default behaviour that can be modified somewhere or an issue? I would want it to remain blank, as it is in the input and the output CSV.

**Comment by Wattenberger** [comment, marked as answer]
good catch, thanks for flagging that! Non-numeric values in numeric columns should be handled correctly, now.

**Reply by blink-twice** [reply]
This was helpful. Thank you!
